iocore/net/quic/QUICConnection.h

```diff
@@ -65,4 +65,7 @@ class QUICConnection : public QUICPacketTransmitter, public QUICFrameTransmitter
   virtual QUICPacketNumber largest_acked_packet_number()    = 0;
   virtual void handle_received_packet(UDPPacket *packeet)   = 0;
   virtual bool is_closed()                                  = 0;
+
+  LINK(QUICConnection, closed_link);
+  SLINK(QUICConnection, closed_alink);
```

Comment:
Is it possible to move these to QUICNetVConnection for now? Would it be a big change?

If it would be a big change, I'm going to accept these for now and merge this PR but I still think these shouldn't be here.